zap/src/test/java/org/parosproxy/paros/core/scanner/AbstractPluginUnitTest.java

```diff
@@ -63,6 +66,14 @@
 /** Unit test for {@link AbstractPlugin}. */
 class AbstractPluginUnitTest extends PluginTestUtils {
 
+    private static final List<String> METHODS_NO_ENCLOSED_CONTENT =
```

Comment:
Move to `AbstractPlugin` and make package-private?

Reply:
I don't want to use a collection there, less garbage as is.